packages/driver/cypress/integration/e2e/multi-domain/multi_domain_config_env_spec.ts

```diff
@@ -0,0 +1,569 @@
+// @ts-ignore / session support is needed for visiting about:blank between tests
+describe('multi-domain - Cypress.config()/Cypress.env()', { experimentalSessionSupport: true, experimentalMultiDomain: true }, () => {
+  beforeEach(() => {
```

Comment:
Since `env` and `config` pretty much follow the same testing pattern, I can probably condense these tests via something like a `forEach` and just run the same tests for `Cypress.env()` and `Cypress.config()`. Any thoughts on this?

Reply:
I think I am going to go ahead and try to clean some of these tests up. I find them a bit confusing and heavily duplicated

Reply:
I wound up deduping these tests in 2eafdee614781cf32b59e18484fc85520bf8be20 with a `forEach`. If this method seems overly complicated, we can always revert!